pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/model/MappedClosure.groovy

```diff
@@ -50,6 +50,14 @@ public abstract class MappedClosure<O,M extends MappedClosure<O,M>> implements S
         this.resultMap.putAll(inMap)
     }
 
+    public O remove(String p) {
+        return resultMap.remove(p)
+    }
+
+    public add(String k, O v) {
```

Comment:
Isn't [put](https://docs.oracle.com/javase/7/docs/api/java/util/Map.html#put(K,%20V)) the more conventional method to add stuff to a map?

Reply:
i r stupid.